client/stylesheets/_toCat.scss

```diff
@@ -4511,3 +4511,140 @@ On click the images will be displayed at normal size to complete the effect
 .profileSettingsButton input:hover {
     background-color: #f73c37 !important;
 }
+
+/* View Feedback */
+
+.profileUserFeedbackView {
+    margin: 0 !important;
+    margin: auto !important;
+    width: 60%;
+    max-width: 1100px;
+}
+
+.profileUserFeedbackViewSymbols {
+    text-align: center;
+    font-size: 3.5rem;
+    color: #989898;
+    margin: 40px auto;
+}
+
+.profileUserFeedbackViewSymbols li {
+    display: inline-block;
+    vertical-align: middle;
+    margin: 0px 40px;
+}
+
+.profileUserFeedbackViewSymbols li i {
+    vertical-align: middle;
+    margin-right: 15px;
+    color: #989898;
+}
+
+.profileUserFeedbackViewPositive {
+    margin-left: 0 !important;
+}
+
+.profileUserFeedbackViewNegative {
+    margin-right: 0 !important;
+}
+
+.profileUserFeedbackViewPositive, .profileUserFeedbackViewPositive i, .profileUserFeedbackViewPositive i:hover {
+    color: #396 !important;
+}
+
+.profileUserFeedbackViewNegative, .profileUserFeedbackViewNegative i, .profileUserFeedbackViewNegative i:hover {
+    color: tomato !important;
+}
+
+.profileUserFeedbackViewCriteria li ul > li {
+    display: inline-block;
+}
+
+.profileUserFeedbackViewCriteria li > ul {
+    font-size: 1.2rem;
+}
+
+.profileUserFeedbackViewCriteria > li {
+     margin: 25px 0;
+     display: inline-block;
+}
+
+.profileUserFeedbackViewQuestion {
```

Comment:
Selector `profileUserFeedbackViewQuestion` should be written in lowercase with hyphens